tests/device_tests/cardano/test_slip39.py

```diff
@@ -146,3 +146,36 @@ def input_flow():
         )
         assert response.tx_hash.hex() == tx_hash
         assert response.tx_body.hex() == tx_body
+
+
+@pytest.mark.altcoin
+@pytest.mark.cardano
+@pytest.mark.skip_t1  # T1 support is not planned
+@pytest.mark.skip_ui
+@pytest.mark.setup_client(mnemonic=MNEMONIC_SLIP39_BASIC_20_3of6, passphrase=True)
+def test_single_passphrase_entry(client):
```

Comment:
In this file called `test_slip39.py` we also have `test_cardano_sign_tx`? That does not seem right